.travis.yml

```diff
@@ -32,7 +33,7 @@ script:
   # Build the onbuild variant because it's FROM the single variant we build
   - docker build -t "${image}-onbuild" "$VERSION/onbuild"
   # Also make sure our deploy script works
-  - coverage run --source=deploy.py ./test_deploy.py && coverage report -m
+  - coverage run --include=deploy.py ./test_deploy.py && coverage report -m
```

Comment:
There were some changes around `--source` and `--include` in the Coverage 4.3 release that I don't fully understand.. but `--source` stopped working for specific files:

```
  --include=PAT1,PAT2,...
                        Include only files whose paths match one of these
                        patterns. Accepts shell-style wildcards, which must be
                        quoted.
...
  --source=SRC1,SRC2,...
                        A list of packages or directories of code to be
                        measured.
```